### DISCUSSION TITLE
Conductor's updated Slack Link?

### DISCUSSION THREAD

**Discussion by prathamjagga** [conductor-oss/conductor#193] [Q&A]
Can anyone provide me the slack URL of conductor slack community. The link mentioned in the repository is expired.

**Comment by v1r3n** [comment]
https://orkes-conductor.slack.com/join/shared_invite/zt-2hmxn0i3n-_W~a9rWMbvMoYmlJo3Y15g#/shared-invite/email

**Comment by rince1013** [comment]
This link is requiring me to have an orkes.io or aerospike.com domain
